docs/apache-airflow-providers-amazon/auth-manager/setup/identity-center.rst

```diff
@@ -0,0 +1,142 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+=================================
+Configure AWS IAM Identity Center
+=================================
+
+In order to use the AWS auth manager, you first need to configure `AWS IAM Identity Center <https://aws.amazon.com/iam/identity-center/>`_.
+AWS IAM Identity Center is used by the AWS auth manager for authentication purposes (login and logout).
+It is then through the AWS IAM Identity Center that the Airflow environment admin manages users and groups.
```

Comment:
I think this can be simplified down, the first time I read this sentence I was a bit lost.
```suggestion
The Airflow environment administrator then uses Identity Center to manage users and groups.
```